min/lib/Minify/Controller/MinApp.php

```diff
@@ -136,13 +136,14 @@ public function setupSources($options) {
                 $base = '/';
             }
             $allowDirs = array();
+            $document_root = realpath($_SERVER['DOCUMENT_ROOT']);
```

Comment:
I don't want to mess with process in 2.x. See if the master branch fixes whatever issue you're having.